frontend/src/components/PackageGeneration/Editor.js

```diff
@@ -35,9 +35,25 @@ class editor extends React.Component {
     }
 
    componentDidMount() {
-    this.setState({code: localStorage.getItem("packageConfigYAML")})
-    this.setState({title: JSON.parse(localStorage.getItem("packageConfigJSON"))["bundle"]["title"]})
-    this.setState({description: JSON.parse(localStorage.getItem("packageConfigJSON"))["bundle"]["description"]})
+
+    if (localStorage.getItem("packageConfigYAML")) {
+      this.setState({code: localStorage.getItem("packageConfigYAML")})
+    } else {
+      this.setState({code: "Kindly generate or paste your configuration here"})
+    }
+
+    if (localStorage.getItem("packageConfigJSON")["bundle"] === undefined) {
+      this.setState({title: "No title Specified"})
+    } else {
+      this.setState({title: JSON.parse(localStorage.getItem("packageConfigJSON"))["bundle"]["title"]})
+    }
+
+    if (localStorage.getItem("packageConfigJSON")["bundle"] === undefined) {
```

Comment:
This is exactly the same comparison as on line 45. Should it also the description? What if the bundle is null?

Reply:
Yeah added an extra condition to fix it . Thanks